### PR TITLE
Ansible 2.x manual

### PR DESCRIPTION
**Pull request by dLobatog**
Some features, [like the section in](https://github.com/theforeman/foreman_ansible/pull/124)   4.6 \{\{ command \}\}. or the [new Remote Execution page](https://github.com/theforeman/foreman_remote_execution/pull/130) need to get merged & packaged.

### CODE REVIEW

File: plugins/foreman_ansible/2.x/index.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+You can use the dynamic inventories feature in Ansible to make your inventory be Foreman, and have host properties and parameters available as Ansible variables in your manifests.
+
+The project is not packaged, but you can find the sources here at [ansible/ansible](https://github.com/ansible/ansible/blob/devel/contrib/inventory/foreman.py). You will need to setup a config file `/etc/ansible/foreman.ini`, which you can find in [ansible/ansible](https://github.com/ansible/ansible/blob/devel/contrib/inventory/foreman.ini).
+
+Check Ansible's [dynamic inventory documentation](http://docs.ansible.com/ansible/intro_dynamic_inventory.html) to install it.
+
+# 3. Configuration
+
+In Foreman, you should add whatever hosts you want to submit facts **from** to the setting `trusted_puppetmaster_hosts`. Change it at Administer > Settings, Auth tab. e.g: If you're running Ansible from host 'A', which SSHs into host 'B', you need to add host 'A'
```
Comment (ares) [inline plugins/foreman_ansible/2.x/index.md]: is this needed if proxy has the right certificate?

**Review by ares — COMMENTED** [review]
looks good, one nit (not a blocker), if you care you can comment out the feature that is still not there, though I'd say it'll be added soon anyway so I'd keep it here

File: plugins/foreman_ansible/2.x/index.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+![schedule ansible job](static/images/plugins/foreman_ansible/schedule_ansible_job.gif)
+
+For more information about scheduled jobs, check out the [Foreman Remote Execution manual](/plugins/foreman_remote_execution)
+
+## 4.5 Overriding Ansible Variables
+
+To override an Ansible variable, set the variable you want to override as a Host Parameter, Host Group Parameter, or Global Parameter. The 'name' of the parameter would be the name of the Ansible variable you want to override, and the 'value' of the parameter the value of the variable.
+
+<!--
```
Comment (dLobatog) [inline plugins/foreman_ansible/2.x/index.md:173]: @ares It seems that this text is still rendered even though it's commented

Comment (ares) [reply plugins/foreman_ansible/2.x/index.md:173]: it's rendered but I don't see it in browser, so it's fine

**Comment by ares** [comment]
Thanks @dLobatog, merging!

**Pull request merged**